src/widgets/barchart/bar.rs

```diff
@@ -41,13 +41,35 @@ pub struct Bar<'a> {
 }
 
 impl<'a> Bar<'a> {
+    /// Creates a new ['Bar'] widget with the given label and value.
+    ///
+    /// The ```label``` parameter accepts any type that is convertible into a [`Line`].
+    /// The ```value``` parameter is the value to be displayed on the bar.
+    ///
+    /// # Examples
+    ///
+    /// ```rust
+    /// # use ratatui::{prelude::*, widgets::*};
+    /// Bar::new("label", 10);
+    /// ```
+    pub fn new<L>(label: L, value: u64) -> Self
+    where
+        L: Into<Line<'a>>,
+    {
+        Self {
+            value,
+            label: Some(label.into()),
+            ..Default::default()
+        }
+    }
+
     /// Set the value of this bar.
     ///
     /// The value will be displayed inside the bar.
     ///
     /// # See also
     ///
-    /// [`Bar::value_style`] to style the value.  
+    /// [`Bar::value_style`] to style the value.
     /// [`Bar::text_value`] to set the displayed value.
```

Comment:
Suggestion to https://github.com/ratatui-org/ratatui/pull/936/files#r1482786822
```suggestion
    /// - [`Bar::value_style`] to style the value.
    /// - [`Bar::text_value`] to set the displayed value.
```